templates/base/generators/component/class.js.hbs

```diff
@@ -0,0 +1,27 @@
+/**
+ *
+ * {{ properCase name }}
+ */
+// @flow
+
+// Node_Module imports
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+
+// Relative imports
+import styles from './{{ properCase name}}.style';
+import type { Props } from './types';
+
+class {{ properCase name }} extends PureComponent<Props> {
+  static defaultProps = {};
+  render() {
+    const {className} = this.props;
+    return <div className={className}/>;
+  }
+}
+
+export default styled({{ properCase name }})`
```

Comment:
This syntax throws an error in flow, for example see below component 

https://github.com/pagesource/atomic-react-components/blob/develop/lib/components/molecules/Accordion/Accordion.js